src/nuklear_font.c

```diff
@@ -197,7 +197,9 @@ nk_font_bake_pack(struct nk_font_baker *baker,
         } while ((it = it->n) != config_iter);
     }
     *height = 0;
-    *width = (total_glyph_count > 1000) ? 1024 : 512;
+    if(max_pack_width) *width = max_pack_width;
+    else *width = (total_glyph_count > 1000) ? 1024 : 512;
+    if(max_pack_height) max_height = max_pack_height;
```

Comment:
A little too packed for my liking. Maybe something like this would improve the readability a little:
```c
*width = max_pack_width
    ? max_pack_width
    : (total_glyph_count > 1000) ? 1024 : 512;

if (max_pack_height)
    max_height = max_pack_height;
```

Reply:
@Hejsil Makes sense.
I really like your double use of the ternary operator here to avoid that if else.
Very creative, will steal :]